qlog/qlog.go

```diff
@@ -17,6 +18,27 @@ import (
 	"github.com/francoispqt/gojay"
 )
 
+// Setting of this only works when quic-go is used as a library.
+// When building a binary from this repository, the version can be set using the following go build flag:
+// -ldflags="-X github.com/lucas-clemente/quic-go/qlog.quicGoVersion=foobar"
+var quicGoVersion = "(devel)"
+
+func init() {
```

Comment:
this func should be a no-op if quickGoVersion is not (devel). otherwise you can overwrite the explicit ldflags value with the implicit runtime/debug version.

Reply:
Fixed.